shell/platform/darwin/ios/framework/Source/FlutterEngineGroupTest.mm

```diff
@@ -37,11 +37,16 @@ - (void)testSpawn {
 }
 
 - (void)testDeleteLastEngine {
+  __weak FlutterEngine* weakSpawner;
   FlutterEngineGroup* group = [[FlutterEngineGroup alloc] initWithName:@"foo" project:nil];
   @autoreleasepool {
     FlutterEngine* spawner = [group makeEngineWithEntrypoint:nil libraryURI:nil];
+    weakSpawner = spawner;
     XCTAssertNotNil(spawner);
+    XCTAssertNotNil(weakSpawner);
   }
+  XCTAssertNil(weakSpawner);
```

Comment:
I could probably get rid of this additional check, but it's arguably useful for avoiding any future accidents wherein someone updates the implementation to hold a strong reference instead of a weak reference that retains the engine. :/